src/freenet/support/MultiValueTable.java

```diff
@@ -73,7 +75,7 @@ public boolean containsElement(K key, V value) {
 			v = table.get(key);
 		}
         return (v == null ?
-                new LimitedEnumeration<V>(null) :
+                new EmptyEnumeration<V>() :
```

Comment:
I considered doing that, but it would sure cause scope creep for this pull request.

I am planning on rewriting `MultiValueTable` to use `HashMap` and `ArrayList` (or plain arrays) instead of `Hashtable` and `Vector` — this class is a great example of how not to do synchronization after all. That'll be a separate pull request.